src/utils/token-password.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable camelcase */
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+/**
+   * Hash Password Method
+   * @param {string} password
+   * @returns {string} returns hashed password
+   */
+const saltRounds = 10;
+const salt = bcrypt.genSaltSync(saltRounds);
+const hashPassword = password => bcrypt.hashSync(password, salt);
```

Comment:
@oyewoas Also, I found out that unless I bring in the dotenv package here and call the config method on it, I still get the Error: secretOrPrivateKey must have a value error.

I think you should help look into this too.
Thanks, and well done.

Reply:
ok i will, though i dont get that error from here...